recipes/jupyter-sphinx/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyter-sphinx" %}
+{% set package = "jupyter_sphinx" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package }}-{{ version }}.tar.gz"
+  sha256: "b5ba1efdd1488b385de0068036a665932ed93998e40ce3a342c60f0926781fd9"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - ipython
+    - ipywidgets >=7.0.0
+    - nbconvert >=5.5
+    - nbformat
+    - sphinx >=2,<4
+
+test:
+  imports:
+    - jupyter_sphinx
+
+about:
+  home: "https://github.com/jupyter/jupyter-sphinx/"
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: "Sphinx extension for rendering of Jupyter interactive widgets."
+  doc_url: https://jupyter-sphinx.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - akhmerov
```

Comment:
@akhmerov - are you ok being a maintainer?

Reply:
Yep, I confirm.

Reply:
yep so this is good to good from my end (obviously the linter error is incorrect)